NACT/attention_query_utilities/attention_query.py

```diff
@@ -441,85 +441,89 @@ def set_scanpy_object(self, new_data):
         self.data = new_data
 
 
-# TODO: @Oscar: could you please take care of the clean up and docstrin of this
-# method please?
+# TODO: @Ali: could you take a look at this and see if there are any changes needed. 
 
-    def gene_tfidf(self, top_n_genestrings_dict: dict = None):
-        """Calculate TF-IDF values for all genes in a top-n ranked dictionary.
+    def calculate_tfidf(top_n_names, top_n_df_dict, n_genes=25):
```

Comment:
**R:** Don't forget to have a one-line summary of the method (per [SG](https://google.github.io/styleguide/pyguide.html#382-modules))